app/grandchallenge/core/templates/challenge.html

```diff
@@ -33,111 +34,125 @@
     {{ block.super }}
 {% endblock %}
 
-{% block sidebar %}
+{% block topbar %}
     {% if challenge %}
-        <div class="col-12 col-md-3 col-lg-2 mb-3">
-            <ul class="nav nav-pills flex-column">
-                {% for page in pages %}
-                    {% if not page.hidden %}
+        <div class="row mb-3 mx-0 border-bottom">
+            <div class="col-md-10 col-sm-8 col-6 px-0">
+                <div class="nav-tab-dropdown-container">
+                    <a class="d-lg-none btn btn-outline-dark mb-1"
+                       data-toggle="dropdown"
+                       href="#"
+                       role="button"
+                       aria-expanded="false"><i class="fas fa-bars"></i></a>
+                    <ul class="nav nav-tabs border-0">
                         <li class="nav-item">
-                            <a class="nav-link {% if page == currentpage %}active{% endif %}"
-                               href="{{ page.get_absolute_url }}">
-                                {% filter title %}
-                                    {% firstof page.display_title page.title %}
-                                {% endfilter %}
+                            <! -- Todo: Adjust the view_name before merging with master (include the page_title slug)-->
```

Comment:
Is this left in here intentionally?

Reply:
Thanks for catching this! I'm actually not fully sure what the correct view_name would be. It works as it is now, even though the detail view for pages normally requires the page slug to work. Not sure why it works without that, and also not sure how I would say something along the lines of "any page slug" because this is the superordinate Info tab which should remain active no matter which subpage the user is on.